client/src/pages/Profile.js

```diff
@@ -321,8 +361,42 @@ export default function Profile({ authProps, setAuthProps }) {
                         <div className="px-4 py-2">{authProps.user.diet}</div>
                       )}
                     </div>
+                    <FormControl size = "small">
+                      <InputLabel>
+                      {authProps.user.whoDietitian === "false"
+                        ? "Select Dietitian"
+                        : authProps.user.whoDietitian}
+                  </InputLabel>
+                  <Select
+                    value = {dietitian}
+                    label = {
+                      authProps.user.whoDietitian === "false"
+                        ? "Select Dietitian"
+                        : authProps.user.whoDietitian
+                    }
+                    onChange = {(event) => {
+                      setDietitian(event.target.value);
+                    }}
+                  >
+                    {dietitians.map((dietitian) => {
```

Comment:
please try to avoid using a state variable's name as an argument variable. it may cause many confusions and issues.